orchestra/utils/notifications.py

```diff
@@ -84,7 +84,8 @@ def notify_status_change(task, previous_status=None):
         }
 
     _notify_internal_slack_status_change(task, current_worker)
-    _notify_experts_slack_status_change(task, current_worker)
+    if task.project.slack_group_id:
```

Comment:
this line won't--it doesn't use the worker at all. 

Line 132, below, makes sure that there is no @-mention if the worker doesn't have a handle.